src/Kdyby/Doctrine/Mapping/AnnotationDriver.php

```diff
@@ -94,15 +95,20 @@ public function getFileExtensions()
 	 */
 	protected function findAllClasses($path)
 	{
-		$loader = new RobotLoader($this->cache !== NULL ? new ReversedStorageDecorator($this->cache) : new MemoryStorage());
+		$loader = new RobotLoader();
+		$loader->setTempDirectory("$path/temp");
```

Comment:
If I'm reading this correctly then this will create a temp dir in every directory configured for searching doctrine entities, right? The problem is that such directory can be set to read-only in production which is why I can't merge this PR.